### PR TITLE
Move min_max unit tests to slt

### PR DESCRIPTION
**Pull request by xinlifoobar**
## Which issue does this PR close?

Closes #.

## Rationale for this change

This change is part of #10384

### CODE REVIEW

File: datafusion/sqllogictest/test_files/aggregate.slt [hunk 1]
```diff
@@ -2480,6 +2480,399 @@ Select bit_xor(DISTINCT c), arrow_typeof(bit_xor(DISTINCT c))  from t;
 statement ok
 drop table t;
 
+#################
+# Min_Max Begin #
+#################
+statement ok
```
Comment (yyy1000) [inline datafusion/sqllogictest/test_files/aggregate.slt:2486]: I think generally it looks good to me. But could you please add some comments to the sqllogictest?
For each unit test, add a comment beginning with `# `

**Review by jayzhan211 — APPROVED** [review]
👍

File: datafusion/sqllogictest/test_files/aggregate.slt (hunk 1 above)
Comment (jayzhan211) [reply datafusion/sqllogictest/test_files/aggregate.slt:2486]: It is nice to have. I fine if there is no comment for trivial test

Comment (xinlifoobar) [reply datafusion/sqllogictest/test_files/aggregate.slt:2487]: Done.

**Comment by jayzhan211** [comment]
Thanks @xinlifoobar and @yyy1000

**Pull request merged**